90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -586,12 +586,7 @@ draw_snapshots() {
 
   zdebug "snapshots: ${snapshots[*]}"
 
-  multi=( "--multi" )
-
-  # skim's --multi doesn't support any arguments
-  if [ "${FUZZYSEL}" == "fzf" ]; then
-    multi+=( "2" )
-  fi
+  multi=( "--multi" "2" )
```

Comment:
Just expand the array where it's used below, it will be clearer.

Reply:
doh, yes.